app/views/photos/home.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="container-fluid" style="padding-top: 20px; padding-left: 30px">
+	<br>
+	<h1 class="welcome">Welcome to the WICI Website!</h1><br><br>
+
+<div class="container-fluid">
+  <div class="row">
+    <% @photos.each do |photo| %>
+      <div class="col-4 home-image" onmouseover="showCaption(this)" onmouseleave="hideCaption(this)">
+        <%= image_tag photo.image, class: "image-size" %>
```

Comment:
images should be cropped, not stretched to fill

Reply:
I'd also recommend enclosing them in a `<div class="col-lg-8 offset-lg-2`, will create some negative space on the side and avoid feeling cramped.